src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/electric/machines/enchanting/AutoDisenchanter.java

```diff
@@ -55,10 +55,10 @@ public ItemStack getProgressBar() {
     @Override
     protected MachineRecipe findNextRecipe(BlockMenu menu) {
         for (int slot : getInputSlots()) {
-            ItemStack item = menu.getItemInSlot(slot);
+            ItemStack item = menu.getItemInSlot(slot == getInputSlots()[0] ? getInputSlots()[1] : getInputSlots()[0]);
```

Comment:
Why is item and secondItem swapped here?

Reply:
No reason actually.... Just to follow AutoEnchanter.java styling.

Reply:
Reverted.